src/Lecture1.hs

```diff
@@ -132,8 +132,9 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 🕯 HINT: Use recursion to implement this function.
 -}
 lowerAndGreater :: Int -> [Int] -> [Char]
-lowerAndGreater n list = show n ++ " is greater than " ++ show greater
-  ++ " elements and lower than " ++ show lower ++ " elements"
-  where	greater	= length (filter ( < n ) list)
-        lower	= length (filter ( > n ) list)
-
+lowerAndGreater n list = output n list (0::Int) (0::Int)
+  where output num numlist greater lower
```

Comment:
I see that the number `n` that you pass in recursive calls doesn't change. You always pass the same number. In that case, you can remove this extra argument and use `n` from the function instead. It's possible because functions in `where` can see all the arguments of the top-level function 🔍

Reply:
Sure! Thanks for the tip!